LayoutTests/fast/dom/document-contentType-DOMParser.html

```diff
@@ -80,6 +80,13 @@
 shouldThrow('new DOMParser().parseFromString(xmlContent, "text/XML").contentType', "'TypeError: Type error'");
 shouldThrow('new DOMParser().parseFromString(htmlContent, "TEXT/html").contentType', "'TypeError: Type error'");
 
+var htmlContentWithSpace = "\n        <div class=\"bpx-player-container\"></div>";
+shouldBeEqualToString('new DOMParser().parseFromString(htmlContentWithSpace, "text/html").body.firstChild.nodeName', 'DIV');
+
+var body = document.createElement("body");
```

Comment:
Please use a `<div>`. I don't think it uses the HTML fast parser if the parent is a `<body>` element.